packages/analytics-browser/src/index.ts

```diff
@@ -9,6 +9,7 @@ export {
   init,
   logEvent,
   remove,
+  reset,
```

Comment:
If `reset` is a public API, you might need to add it into the snippet: https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-browser/scripts/templates/browser-snippet.template.js#L78-L89

Reply:
Thanks for reviewing! Good catch, updated!